addons/resource/views/resource_views.xml

```diff
@@ -213,6 +213,7 @@
         <field name="model">resource.calendar.attendance</field>
         <field name="arch" type="xml">
             <form string="Working Time">
+                <sheet>
```

Comment:
Why not group name below ? (As you are already touching this)

Reply:
As it is just one group on the page. No group inside a group or anything either so the chances of this changing are very low and the XPath is pretty clean for this. Plus, @mart-e wouldn't be happy I bet. ;)